i cant download 560.35.03

i cant download  560.35.03, when its going to be released? please i need it please just put it

The only reason 560.35.03 is not on Flathub yet is because the i386 driver [is failing to build](https://buildbot.flathub.org/#/builders/44/builds/29/steps/7/logs/stdio) because of a CMake/Ninja-related error:

<details><summary>Expand</summary>

```
========================================================================
Building module libarchive in /srv/buildbot/worker/build-i386-2/build/.flatpak-builder/build/libarchive-1
========================================================================
]2;flatpak-builder: Building libarchiveFB: Running: tar xf /srv/buildbot/worker/build-i386-2/build/.flatpak-builder/downloads/f887755c434a736a609cbd28d87ddbfbe9d6a3bb5b703c22c02f6af80a802735/libarchive-3.7.4.tar.xz --no-same-owner --strip-components=1 -J
FB: Running: flatpak build --die-with-parent --env=FLATPAK_BUILDER_BUILDDIR=/run/build/libarchive --nofilesystem=host:reset --filesystem=/srv/buildbot/worker/build-i386-2/build/.flatpak-builder/build/libarchive-1 --bind-mount=/run/build/libarchive=/srv/buildbot/worker/build-i386-2/build/.flatpak-builder/build/libarchive-1 --build-dir=/run/build/libarchive --bind-mount=/run/ccache=/srv/buildbot/worker/build-i386-2/build/.flatpak-builder/ccache --unshare=network --env=SOURCE_DATE_EPOCH=1724275480 '--env=CFLAGS=-O2 -g -I/usr/lib/GL/nvidia-560-31-02/include' '--env=CXXFLAGS=-O2 -g' --env=LDFLAGS=-L/usr/lib/GL/nvidia-560-31-02/lib --env=V=1 --env=CCACHE_DIR=/run/ccache --env=PATH=/run/ccache/bin:/app/bin:/usr/bin --env=LD_LIBRARY_PATH=/app/lib:/usr/lib/GL/nvidia-560-31-02/lib --env=PKG_CONFIG_PATH=/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/GL/nvidia-560-31-02/lib/pkgconfig --env=FLATPAK_BUILDER_N_JOBS=64 /srv/buildbot/worker/build-i386-2/build/.flatpak-builder/rofiles/rofiles-VQgSi2 cmake '-DCMAKE_INSTALL_PREFIX:PATH='\''/usr/lib/GL/nvidia-560-31-02'\''' -G Ninja . -DBUILD_SHARED_LIBS=OFF -DENABLE_OPENSSL=OFF -DENABLE_LIBB2=OFF -DENABLE_LZ4=OFF -DENABLE_BZip2=OFF -DENABLE_LIBXML2=OFF -DENABLE_EXPAT=OFF -DENABLE_CNG=OFF -DENABLE_TAR=OFF -DENABLE_CPIO=OFF -DENABLE_CAT=OFF -DENABLE_UNZIP=OFF -DENABLE_XATTR=OFF -DENABLE_ACL=OFF -DENABLE_ICONV=OFF -DENABLE_TEST=OFF
CMake Error at /usr/share/cmake-3.7/Modules/CMakeDetermineCompilerId.cmake:131 (file):
  file problem creating directory:
  /run/build/libarchive/CMakeFiles/3.7.2/CompilerIdC
Call Stack (most recent call first):
  /usr/share/cmake-3.7/Modules/CMakeDetermineCompilerId.cmake:30 (CMAKE_DETERMINE_COMPILER_ID_BUILD)
  /usr/share/cmake-3.7/Modules/CMakeDetermineCCompiler.cmake:112 (CMAKE_DETERMINE_COMPILER_ID)
  CMakeLists.txt:13 (PROJECT)
CMake Error at /usr/share/cmake-3.7/Modules/CMakeDetermineCompilerId.cmake:131 (file):
  file problem creating directory:
  /run/build/libarchive/CMakeFiles/3.7.2/CompilerIdC
Call Stack (most recent call first):
  /usr/share/cmake-3.7/Modules/CMakeDetermineCompilerId.cmake:30 (CMAKE_DETERMINE_COMPILER_ID_BUILD)
  /usr/share/cmake-3.7/Modules/CMakeDetermineCCompiler.cmake:112 (CMAKE_DETERMINE_COMPILER_ID)
  CMakeLists.txt:13 (PROJECT)
CMake Error at /usr/share/cmake-3.7/Modules/CMakeDetermineCompilerId.cmake:131 (file):
  file problem creating directory:
  /run/build/libarchive/CMakeFiles/3.7.2/CompilerIdC
Call Stack (most recent call first):
  /usr/share/cmake-3.7/Modules/CMakeDetermineCompilerId.cmake:30 (CMAKE_DETERMINE_COMPILER_ID_BUILD)
  /usr/share/cmake-3.7/Modules/CMakeDetermineCCompiler.cmake:112 (CMAKE_DETERMINE_COMPILER_ID)
  CMakeLists.txt:13 (PROJECT)
-- The C compiler identification is unknown
-- Check for working C compiler: /run/ccache/bin/cc
-- Check for working C compiler: /run/ccache/bin/cc -- broken
CMake Error at /usr/share/cmake-3.7/Modules/CMakeTestCCompiler.cmake:51 (message):
  The C compiler "/run/ccache/bin/cc" is not able to compile a simple test
  program.
  It fails with the following output:
   Change Dir: /run/build/libarchive/CMakeFiles/CMakeTmp
  
  Run Build Command:"/usr/bin/ninja" "cmTC_5fec9"
  ninja: error: rebuilding 'build.ninja': stat(build.ninja): Value too large
  for defined data type
  
  
  CMake will not be able to correctly generate this project.
Call Stack (most recent call first):
  CMakeLists.txt:13 (PROJECT)
-- Configuring incomplete, errors occurred!
See also "/run/build/libarchive/CMakeFiles/CMakeOutput.log".
See also "/run/build/libarchive/CMakeFiles/CMakeError.log".
]2;flatpak-builder: Cleanup libarchiveError: module libarchive: Child process exited with code 1
FB: Unmounting read-only fs: fusermount -uz /srv/buildbot/worker/build-i386-2/build/.flatpak-builder/rofiles/rofiles-VQgSi2
```
</details>

I think the error is caused by https://github.com/ninja-build/ninja/issues/829 and it's been solved already, but we can't really upgrade Ninja, so we might have to revert back [this change](https://github.com/flathub/org.freedesktop.Platform.GL.nvidia/commit/b1d0cafed1172544fa7ff4049a0ff12657a3de00) and go back to Autotools.

I'll open a pull request reverting that change, and hopefully Flathub should be able to build the i386 driver.

--- a/org.freedesktop.Platform.GL.nvidia.json.in
+++ b/org.freedesktop.Platform.GL.nvidia.json.in
@@ -10,6 +10,7 @@
     "cleanup": ["/include"],
     "build-options" : {
         "append-ld-library-path": "/usr/lib/GL/nvidia-@@NVIDIA_VERSION@@/lib",
+        "append-pkg-config-path": "/usr/lib/GL/nvidia-@@NVIDIA_VERSION@@/lib/pkgconfig",
         "cflags": "-O2 -g -I/usr/lib/GL/nvidia-@@NVIDIA_VERSION@@/include",
         "cxxflags": "-O2 -g",
         "prefix": "/usr/lib/GL/nvidia-@@NVIDIA_VERSION@@",
@@ -48,7 +49,7 @@
             "config-opts": [ "--disable-shared", "--enable-static", "--disable-xattr", "--disable-acl",
                              "--without-bz2lib", "--without-iconv", "--without-lz4", "--without-lzo2", "--without-nettle",
                              "--without-openssl", "--without-xml2", "--without-expat", "--disable-bsdcat", "--disable-bsdcpio",
-                             "--disable-bsdtar"
+                             "--disable-bsdtar", "--disable-bsdunzip", "--without-libb2", "--without-cng", "--without-mbedtls"
                            ],
             "sources": [
                 {

--- a/org.freedesktop.Platform.GL.nvidia.json.in
+++ b/org.freedesktop.Platform.GL.nvidia.json.in
@@ -10,7 +10,6 @@
     "cleanup": ["/include"],
     "build-options" : {
         "append-ld-library-path": "/usr/lib/GL/nvidia-@@NVIDIA_VERSION@@/lib",
-        "append-pkg-config-path": "/usr/lib/GL/nvidia-@@NVIDIA_VERSION@@/lib/pkgconfig",
         "cflags": "-O2 -g -I/usr/lib/GL/nvidia-@@NVIDIA_VERSION@@/include",
         "cxxflags": "-O2 -g",
         "prefix": "/usr/lib/GL/nvidia-@@NVIDIA_VERSION@@",
@@ -44,27 +43,13 @@
             ]
         },
         {
-            "cleanup": ["*"],
+            "cleanup": ["/include", "/share"],
             "name": "libarchive",
-            "buildsystem": "cmake-ninja",
-            "config-opts": [
-                "-DBUILD_SHARED_LIBS=OFF",
-                "-DENABLE_OPENSSL=OFF",
-                "-DENABLE_LIBB2=OFF",
-                "-DENABLE_LZ4=OFF",
-                "-DENABLE_BZip2=OFF",
-                "-DENABLE_LIBXML2=OFF",
-                "-DENABLE_EXPAT=OFF",
-                "-DENABLE_CNG=OFF",
-                "-DENABLE_TAR=OFF",
-                "-DENABLE_CPIO=OFF",
-                "-DENABLE_CAT=OFF",
-                "-DENABLE_UNZIP=OFF",
-                "-DENABLE_XATTR=OFF",
-                "-DENABLE_ACL=OFF",
-                "-DENABLE_ICONV=OFF",
-                "-DENABLE_TEST=OFF"
-            ],
+            "config-opts": [ "--disable-shared", "--enable-static", "--disable-xattr", "--disable-acl",
+                             "--without-bz2lib", "--without-iconv", "--without-lz4", "--without-lzo2", "--without-nettle",
+                             "--without-openssl", "--without-xml2", "--without-expat", "--disable-bsdcat", "--disable-bsdcpio",
+                             "--disable-bsdtar"
+                           ],
             "sources": [
                 {
                     "type": "archive",

